tfx/components/base/base_executor.py

```diff
@@ -59,6 +59,10 @@ def get_tmp_path(self) -> Text:
         raise RuntimeError('Temp path not available')
       return os.path.join(self._tmp_dir, str(self._unique_id), '')
 
+    @property
+    def unique_id(self):
+      return self._unique_id
```

Comment:
Can we avoid adding this to the BaseExecutor class and just access the `._unique_id` field?  I think this might be better since we reduce changes to the externally visible API until users need it.

Reply:
Code reverted.